.travis.yml

```diff
@@ -11,6 +11,7 @@ cache:
 
 install:
     - pip install pipenv
+    - pip install --upgrade setuptools
```

Comment:
If needed, this can probably sit on the line above: 
`- pip install -U pipenv setuptools`

Reply:
I've removed this now